assets/src/components/with-cropped-featured-image.js

```diff
@@ -255,7 +251,7 @@ export default ( InitialMediaUpload ) => {
 				data.height = height;
 			}
 
-			onSelect( data );
+			onSelect( data ); // @todo Does this do anything?
```

Comment:
Well it passes the information to whatever `onSelect` callback prop that was passed to the component.